### DISCUSSION TITLE
0.13 Release

### DISCUSSION THREAD

**Discussion by sjtrny** [blakeblackshear/frigate#7356] [General]
Hi, is there an estimated date or date range for 0.13 release?

**Comment by AlexM137** [comment]
No

**Comment by tomber71** [comment]
This year?

**Comment by NickM-27** [comment]
Soon™

**Reply by tomber71** [reply]
Have plans to replace my fathers ZoneMinder this month and was hoping for a 0.13 release soon. This is the best. GOOD work all!!!
Running HAOS and frigate as an add-on. Coral M.2 and HP DeskPro and 5 cams.

**Reply by NickM-27** [reply]
There will likely be many betas before the full release

**Reply by tomber71** [reply]
Ok. Thanks for the info. Deliver stable 0.12 is the right way for production use when i thing about it.

**Comment by sjtrny** [comment]
Thanks @NickM-27, looking forward to it and appreciate the work you've put into Frigate.

**Comment by joelegg** [comment]
awesome work Nick. thank you

**Comment by dopeytree** [comment]
What's the correct tag to swap to  v0.13.0-7C629C1 in docker?

ghcr.io/blakeblackshear/frigate:stable is the default and I've tried putting beta instead of stable

**Reply by NickM-27** [reply]
packages are available at https://github.com/blakeblackshear/frigate/pkgs/container/frigate but be sure to read the preview docs before moving, there are breaking changes (or just wait for the beta)

**Reply by dopeytree** [reply]
ok thanks yeah I'll hold out for the beta

**Reply by h2rSys** [reply]
where are the preview docs to read??

**Comment by tomber71** [comment]
No release in 2023? Maybe in 2024 then ;-)

**Reply by moandcompany** [reply]
13.0 Release Candidate 1 is out :)

**Reply by blalor** [reply]
looks like rc1 may have been pulled?

**Reply by NickM-27** [reply]
no, it has not been pulled, some users just aren't seeing it in the release for some reason

**Comment by HIhnen** [comment]
> where are the preview docs to read??

https://deploy-preview-6262--frigate-docs.netlify.app
